src/parser/test-helpers.ts

```diff
@@ -45,6 +45,18 @@ export const buildParser = () => {
   };
 };
 
+export const buildPreprocessorParser = () => {
+  execSync(
+    'npx peggy --cache --format es -o src/preprocessor/preprocessor-parser.js src/preprocessor/preprocessor-grammar.pegjs'
+  );
+  const parser = require('../preprocessor/preprocessor-parser');
+  const parse = parser.parse as Parse;
+  return {
+    parse,
+    parser,
+  };
+};
```

Comment:
this build change is to support the plumbing to make the `import` statement work in `preprocessor-grammar.pegjs` below, as peggyjs does not support typescript imports